engine/src/main/java/io/camunda/zeebe/engine/api/ProcessingScheduleService.java

```diff
@@ -13,7 +13,9 @@
 
 public interface ProcessingScheduleService {
 
-  void runDelayed(Duration delay, Runnable followUpTask);
+  void runDelayed(Duration delay, Runnable task);
```

Comment:
:question: Can we remove this now?

Reply:
I think the whole interface needs an overhaul. I pretty much added methods so I could migrate existing code. And then I was hoping you would look at this mess and clean up the interface at some point.

Reply:
Alright I will do this then :) :+1: